Files/Files.csproj

```diff
@@ -190,6 +190,8 @@
     <Compile Include="Enums\FilesystemOperationType.cs" />
     <Compile Include="Enums\FolderLayout.cs" />
     <Compile Include="Enums\FolderLayoutModes.cs" />
+    <Compile Include="EventArguments\BundlesLoadIconOverlayEventArgs.cs" />
+    <Compile Include="EventArguments\BundlesOpenPathEventArgs.cs" />
```

Comment:
Should these be put in a sub folder for bundles?

Reply:
@yaichenbaum all done!